src/itemLogin/factory.ts

```diff
@@ -0,0 +1,14 @@
+import { ItemLoginSchema, ItemLoginSchemaType } from './itemLogin.js';
+import { faker } from '@faker-js/faker';
+
+export function itemLoginSchemaFactory(
```

Comment:
Is this should be `ItemLoginSchemaFactory` ? I saw that the account factory is in this style.